### PR TITLE
Update transformers.py

### PR DESCRIPTION
**Pull request by freunda**
fix for #546

### CODE REVIEW

**Review by delliott90 — APPROVED** [review]

File: stix_shifter_modules/arcsight/stix_translation/transformers.py [hunk 1]
```diff
@@ -14,8 +14,9 @@ def transform(registry_key):
                                   "HKPD": "HKEY_PERFORMANCE_DATA", "HKU": "HKEY_USERS", "HKDD": "HKEY_DYN_DATA"}
         try:
             root_key = registry_key.split("\\")
-            map_root_key = stix_root_keys_mapping[root_key[0]]
-            root_key[0] = map_root_key
+            if splited[0] in stix_root_keys_mapping:
+                map_root_key = stix_root_keys_mapping[root_key[0]]
```
Comment (mdazam1942) [inline stix_shifter_modules/arcsight/stix_translation/transformers.py:18]: should this be `map_root_key = stix_root_keys_mapping[splited[0]]`

Comment (mdazam1942) [inline stix_shifter_modules/arcsight/stix_translation/transformers.py:18]: should this be map_root_key = stix_root_keys_mapping[splited[0]]

**Review by mdazam1942 — CHANGES_REQUESTED** [review]

File: stix_shifter_modules/arcsight/stix_translation/transformers.py [hunk 2]
```diff
@@ -14,8 +14,9 @@ def transform(registry_key):
                                   "HKPD": "HKEY_PERFORMANCE_DATA", "HKU": "HKEY_USERS", "HKDD": "HKEY_DYN_DATA"}
         try:
             root_key = registry_key.split("\\")
-            map_root_key = stix_root_keys_mapping[root_key[0]]
-            root_key[0] = map_root_key
+            if splited[0] in stix_root_keys_mapping:
```
Comment (delliott90) [inline stix_shifter_modules/arcsight/stix_translation/transformers.py:17]: `splited` is never defined. It's causing the builds to fail. Needs to be set to `root_key` in this case.

**Review by delliott90 — CHANGES_REQUESTED** [review]

**Comment by freunda** [comment]
fixed unreferenced split

**Review by mdazam1942 — APPROVED** [review]

**Pull request merged**